contracts/src/interfaces/Teleporter/IAvalancheICTTRouter.sol

```diff
@@ -9,6 +9,8 @@ struct DestinationBridge {
     address bridgeAddress;
     uint256 requiredGasLimit;
     bool isMultihop;
+    uint256 minimalPrimaryRelayerFee;
+    uint256 minimalSecondaryRelayerFee;
```

Comment:
I think this should be in the `IAvalancheICTTRouterFixedFee` interface since it is not used in `IAvalancheICTTRouter`.

We should probably use a separate storage for this as well. I propose:
```
struct MinBridgeFees {
    uint256 minPrimaryRelayerFee;
    uint256 minSecondaryRelayerFee;
}

mapping(
        bytes32 destinationChainID => mapping(address token => MinBridgeFees minBridgeFees)
    ) public destinationChainTokenToMinBridgeFees;
```

Reply:
So you would set the minimal fee in the constructor of the contract`AvalancheICTTRouterFixedFees`?